vendor/engines/secure_rooms/app/models/secure_room_price_policy.rb

```diff
@@ -2,4 +2,22 @@ class SecureRoomPricePolicy < PricePolicy
 
   include PricePolicies::Usage
 
+  def estimate_cost_and_subsidy_from_order_detail(order_detail)
+    calculate_cost_and_subsidy_from_order_detail(order_detail)
+  end
+
+  def calculate_cost_and_subsidy_from_order_detail(order_detail)
+    entry_at = order_detail.occupancy.entry_at
```

Comment:
I left this as the more pessimistic route of raising an error if `occupancy` is nil. That case _shouldn't_ happen, and if it does it most likely indicates a bug upstream somewhere. This is the kind of thing I'd like to catch early on if we can rather than not raising and returning `nil`s, which would be harder to track down.

If we do end up needing the `try`, we should at that point have found out why and can account for that, or at the least comment on what happened.